src/bin-driver.rs

```diff
@@ -0,0 +1,15 @@
+use anyhow::{anyhow, Result};
```

Comment:
Some module level documentation may be good here to explain the difference between the driver and the wrapper.